fern-aviles/my-design-lite
# Refactored code and fixed duplication on group select

What's New
-
- Moved all files inside my-design to my-design-lite
- Fixed bug that added another rotor whenever the user would try group selecting rotors

Set up
-
- Clone repo
- In your command line:
  - run `npm install` to install necessary packages
  - run `npm run dev` to run the project in your web browser

On your browser, go to `localhost:5173` and you should see a page with the word 'canvas' at the top

Testing
- 
- Create multiple rotors and group select them
  - When group selecting rotors, a new rotor should not appear
  - When moving a group of rotors around, a new rotor should not appear
- Verify there's no `my-design` folder in `my-design-lite`
